SpatialGDK/Source/SpatialGDKEditorToolbar/Private/SpatialGDKEditorToolbar.cpp

```diff
@@ -843,7 +816,7 @@ void FSpatialGDKEditorToolbarModule::ShowSimulatedPlayerDeploymentDialog()
 
 void FSpatialGDKEditorToolbarModule::OpenLaunchConfigurationEditor()
 {
-	ULaunchConfigurationEditor::LaunchTransientUObjectEditor<ULaunchConfigurationEditor>(TEXT("Launch Configuration Editor"));
+	ULaunchConfigurationEditor::LaunchTransientUObjectEditor<ULaunchConfigurationEditor>(TEXT("Launch Configuration Editor"), nullptr);
```

Comment:
How about instead of `nullptr` as the parent window we make the editor window the parent? Wouldn't that mean the window starts being used as a tab and can be pinned around the editor like the OutputLog / Details panels?

Reply:
I can try, but I believe it won't be so easy to get the tabbing behaviour that way.

Reply:
I added that so when opened from the launch sim deployment panel, closing the launch conf editor goes back to the deployment panel, instead of hiding it.

Reply:
Yeah you might need to change `TSharedRef<SWindow> NewSlateWindow = SNew(SWindow)` to be an `SWidget` instead to get tabbing to play nice

Reply:
Or actually, maybe you can make use of `TSharedPtr<SDockTab> FTabManager::SpawnTab(const FTabId& TabId, const TSharedPtr<SWindow>& ParentWindow, const bool bCanOutputBeNullptr)` to add as a widget

Reply:
That's a protected function call, I'll make a follow up ticket.